pkg/hub/hubclient/hubclient.go

```diff
@@ -143,25 +146,33 @@ func PostClusterInfoToHub(ctx context.Context, spokeclient client.Client, hubCli
 }
 
 func PostDashboardCredsToHub(ctx context.Context, spokeclient, hubClient client.Client) error {
-	sa := &corev1.ServiceAccount{
+	tokenReq := &authv1.TokenRequest{
```

Comment:
rbac needs to be added for this!

Reply:
nice find! I completely missed this
Maybe it is easier to close this PR and add the serviceaccount with secret in the helm charts itself

Reply:
> nice find! I completely missed this Maybe it is easier to close this PR and add the serviceaccount with secret in the helm charts itself

makes sense!